examples/hashing-migration.js

```diff
@@ -11,12 +11,12 @@ exports.up = function(done) {
     return new Promise(function(resolve, reject) {
       hash.saltAndHash(user.password, function(err, hashed) {
         if (err) {
-          return reject(err);
+          return reject(new Error(err));
```

Comment:
Verified that this was the only one occurrence where we used strings for reject.